src/requirements/Sound/components/SoundSupportArtist.tsx

```diff
@@ -0,0 +1,81 @@
+import { FormControl, FormLabel } from "@chakra-ui/react"
+import FormErrorMessage from "components/common/FormErrorMessage"
+import StyledSelect from "components/common/StyledSelect"
+import { useState } from "react"
+import { Controller, useFormContext, useWatch } from "react-hook-form"
+import { RequirementFormProps } from "requirements"
+import useSWRImmutable from "swr/immutable"
+import { SelectOption } from "types"
+
+import parseFromObject from "utils/parseFromObject"
+
+const SupportArtist = ({ baseFieldPath }: RequirementFormProps) => {
+  const {
+    control,
+    formState: { errors },
+  } = useFormContext()
+
+  const [search, setSearch] = useState(
+    useWatch({ name: `${baseFieldPath}.data.id` })
+  )
+
+  const { data: artistsData, isValidating: artistsLoading } = useSWRImmutable(
```

Comment:
the artist input and corresponding data fetching is repeated in every type's form, it should be abstracted to a SoundArtistSelect component